src/ReinforcementLearningCore/src/core/stop_conditions.jl

```diff
@@ -58,12 +58,12 @@ function _stop_after_step(s::StopAfterStep)
     res
 end
 
-function check_stop(s::StopAfterStep, args...)
+function check!(s::StopAfterStep, args...)
```

Comment:
Since we are renaming the conditions, should this be renamed to `StopAfterNSteps` to match the naming of the reset condition ?